crates/core/plugin_sm/tests/plugin.rs

```diff
@@ -34,8 +38,35 @@ mod tests {
         assert_eq!(res, Ok(()));
     }
 
+    #[test_case("abc", Some("1.0")  ; "with version")]
+    #[test_case("abc",None  ; "without version")]
```

Comment:
I might be good to also have a test to check that a trailing comma is accepted when there is no version.

Reply:
Do you mean with the trailing tab?. If So, I added a new test.